up-l2/dispatchers/README.adoc

```diff
@@ -30,11 +30,11 @@ SPDX-License-Identifier: Apache-2.0
 ----
 
 
-Like IP packets, uProtocol messages (CE) have a source attribute (originator of the message) and sink attribute (where should the message be sent to). These attributes are used to route the CEs from one uE to the next if the destination is not the receiving uE.
+Like IP packets, uProtocol messages (UMessage) have a source attribute (originator of the message) and sink attribute (where should the message be sent to). These attributes are used to route the CEs from one uE to the next if the destination is not the receiving uE.
 
 NOTE: Dispatcher and router shall be used For the remainder of this document we will use the term dispatcher and router interchangeably.
 
-The header contains information for routing as well as metadata (of the data). One of the core principles of uProtocol is that the data portion of CE *MUST* be untouched by message routers, this is very similar to how most Internet standards work today. Only the source who generated the CE and the sink who will consume the CE needs to understand/analyze the payload of the CE.
+The header contains information for routing as well as metadata (of the data). One of the core principles of uProtocol is that the data portion of UMessage *MUST* be untouched by message routers, this is very similar to how most Internet standards work today. Only the source who generated the UMessage and the sink who will consume the UMessage needs to understand/analyze the payload of the UMessage.
```

Comment:
We can read, but also _should not be_ modifying the UUri and UAttributes in order to perform routing. Not sure if we have to explicitly state that in the spec. Feels like better to be clear on that point tho. WDYT?

Reply:
I'd agree that we shouldn't modify source/destination UURis and existing uAttributes.  We might need to add an attribute in the chain of transport.  

I was going to say in accepting a SOME/IP message into the uProtocol universe there is the need to add an attribute saying whether any MAC protection checks happened and passed.  But that happens at the protocol boundary, so really isn't the same thing.  This might be useful between devices inside the same domain if MAC checks are included and messages not passing the MAC testing on ingestion aren't just dropped.

Is there any transport special "events" that might want us to add a hint to a message as an attribute?  Perhaps messages there were queued from the previous power cycle would benefit from a hint to the destination (if they care to look for it) saying "this message was delayed in being delivered".  This might trigger special handling at the sink.